mlir/lib/Dialect/Arith/IR/ArithDialect.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "mlir/Conversion/ConvertToLLVM/ToLLVMInterface.h"
```

Comment:
Ideally, we shouldn't have something under `lib/Dialect` depend on something in `lib/Conversion`. I see there is precedent, so I can allow this for consistency. But this hits the design problem I'd like to ultimately address: where can we list promises.